### PR TITLE
Ft milestone4: Adding State to App and Event handlers

### PR DESCRIPTION
**Pull request by didymus707**
Here state was added to the App as well as event handlers which were passed down to child components as props

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes Required❌.

# 🙋‍♂️ @didymus707 
#### You have done a good job on this one so far, your code is clean and easy to read. Good job on turning App.js to a class component😇.

* [x] Let us move App.js to container since it has now been converted into a class and has it owns state, it does no longer qualify tot be in the presentational component folder.

* [x] The `%` functionality is not functioning as required, let us address it. When we select percentage, if `num1` and `num2` exist it should resolve the computation and return the results in percentage. If only `num1` exist resolve `num1` to percentage.
ie `2 + 5` then `%` answer: `0.07` or `2` then `%` answer: `0.07`

* [ ] Dividing a number by zero, throws the application into error instead of returning undefined.

* [ ] When I click say `number` then `operator` then `.` the application throws into error, let us address it.

#### That is it kindly make the changes and submit the project for a re-review😊.

#### Happy coding👨‍💻

Addo Forison

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @didymus707  :wave:

__Thank you for implementing some of the changes required by the previous reviewer.
However, we still have some changes to be made before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] You seem to be having a merge conflict in your PR. Kindly fix the conflict. Also, your GitHub Actions checks (ESlint & Stylelint) seems not to be running. Kindly fix them.
conflict

- [ ] Your PR is confusing. You requested for review of Milestone 5 of the React Calculator project, whereas, you raised a PR from `ft-milestone4` to `develop`. Are you requesting a review of milestone4? Did you fill the wrong code review request form? Is it a problem with your naming convention? Kindly make clarifications and fix whatever is necessary.
- Here's your review request, based on the review request form you filled: 
request
- Then, the PR link you submitted directed me to this:
confuse





* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request closed without merging**
